libs/blocks/chart/chart.js

```diff
@@ -263,8 +288,12 @@ const init = async (el) => {
   const data = await fetchData(dataLink);
   if (!data) return;
 
-  // ToDo: Replace colors MWPW-112994
-  const colors = ['red', 'blue', 'green'];
+  const authoredColor = Array.from(chartStyles)?.find(style => style in colorPalette);
```

Comment:
I believe there should be parentheses around the param. Please make sure you have the linter set up correctly

```suggestion
  const authoredColor = Array.from(chartStyles)?.find((style) => style in colorPalette);
```

Reply:
Updated.